### PR TITLE
cmuwmtopbm, escp2topbm, pbmtocmuwm, pbmtoepsi, ebmtoepson, pbmtoescp2, pbmtogo: add pages

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/cmuwmtopbm.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# cmuwmtopbm
+
+> Convert a CMU window manager bitmap into a PBM image.
```
Comment (vitorhcl) [inline pages/common/cmuwmtopbm.md:3]:
```suggestion
> Convert a CMU window manager bitmap to a PBM image.
```
For consistency, shouldn't we prefer "convert ... to ..." over "convert ... into ..."?

File: pages/common/cmuwmtopbm.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+> See also: `pbmtocmuwm`.
+> More information: <https://netpbm.sourceforge.net/doc/cmuwmtopbm.html>.
+
+- Convert a CMU window manager bitmap into a PBM image:
```
Comment (vitorhcl) [inline pages/common/cmuwmtopbm.md:7]:
```suggestion
- Convert a CMU window manager bitmap to a PBM image:
```

File: pages/common/pbmtocmuwm.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# pbmtocmuwm
+
+> Convert a PBM image into a CMU window manager bitmap.
```
Comment (vitorhcl) [inline pages/common/pbmtocmuwm.md:3]:
```suggestion
> Convert a PBM image to a CMU window manager bitmap.
```

File: pages/common/pbmtocmuwm.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+> See also: `cmuwmtopbm`.
+> More information: <https://netpbm.sourceforge.net/doc/pbmtocmuwm.html>.
+
+- Convert a PBM image into a CMU window manager bitmap:
```
Comment (vitorhcl) [inline pages/common/pbmtocmuwm.md:7]:
```suggestion
- Convert a PBM image to a CMU window manager bitmap:
```

File: pages/common/pbmtoepsi.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# pbmtoepsi
+
+> Convert a PBM image into an encapsulated PostScript style preview bitmap.
```
Comment (vitorhcl) [inline pages/common/pbmtoepsi.md:3]:
```suggestion
> Convert a PBM image to an encapsulated PostScript style preview bitmap.
```

File: pages/common/pbmtoepsi.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://netpbm.sourceforge.net/doc/pbmtoepsi.html>.
+
+- Convert a PBM image into an encapsulated PostScript style preview bitmap:
```
Comment (vitorhcl) [inline pages/common/pbmtoepsi.md:6]:
```suggestion
- Convert a PBM image to an encapsulated PostScript style preview bitmap:
```

File: pages/common/pbmtoepson.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# pbmtoepson
+
+> Convert a PBM image into an Epson printer graphic.
+> See also: `pbmtoescp2`.
+> More information: <https://netpbm.sourceforge.net/doc/pbmtoepson.html>.
+
+- Convert a PBM image into an Epson printer graphic:
```
Comment (vitorhcl) [inline pages/common/pbmtoepson.md:7]:
```suggestion
- Convert a PBM image to an Epson printer graphic:
```

File: pages/common/pbmtoepson.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# pbmtoepson
+
+> Convert a PBM image into an Epson printer graphic.
```
Comment (vitorhcl) [inline pages/common/pbmtoepson.md:3]:
```suggestion
> Convert a PBM image to an Epson printer graphic.
```

File: pages/common/pbmtoescp2.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+> Convert a PBM image to a ESC/P2 printer file.
+> See also: `pbmtoepson`, `escp2topbm`.
+> More information: <https://netpbm.sourceforge.net/doc/pbmtoescp2.html>.
+
+- Convert a PBM image to a ESC/P2 printer file:
+
+`pbmtoescp2 {{path/to/image.pbm}} > {{path/to/output.escp2}}`
+
+- Control the compression of the output:
```
Comment (vitorhcl) [inline pages/common/pbmtoescp2.md:11]:
```suggestion
- Specify the compression of the output:
```

File: pages/common/pbmtogo.md [hunk 10]
```diff
@@ -0,0 +1,8 @@
+# pbmtogo
+
+> Convert a PBM image into a compressed GraphOn graphic.
```
Comment (vitorhcl) [inline pages/common/pbmtogo.md:3]:
```suggestion
> Convert a PBM image to a compressed GraphOn graphic.
```

File: pages/common/pbmtogo.md [hunk 11]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://netpbm.sourceforge.net/doc/pbmtogo.html>.
+
+- Convert a PBM image into a compressed GraphOn graphic:
```
Comment (vitorhcl) [inline pages/common/pbmtogo.md:6]:
```suggestion
- Convert a PBM image to a compressed GraphOn graphic:
```

**Review by vitorhcl — COMMENTED** [review]
Thank you for your contribution! I have some suggestions

**Review by kbdharun — APPROVED** [review]

**Pull request merged**